Framework/CurveFitting/CMakeLists.txt

```diff
@@ -78,6 +78,7 @@ set ( SRC_FILES
 	src/Functions/FlatBackground.cpp
 	src/Functions/FullprofPolynomial.cpp
 	src/Functions/FunctionGenerator.cpp
+	src/Functions/FunctionQDepends.cpp
```

Comment:
Initially I placed it on API as you suggest, but then I found out I could not write a unit test because DataObjects (Workspaces) are not built before API. The other sensible place down the make hierarchy was CurveFitting, so I placed it there.